e2etest/pkg/k8s/services.go

```diff
@@ -30,17 +31,22 @@ func GetSvcNode(cs clientset.Interface, svcNS string, svcName string, allNodes *
 		return nil, err
 	}
 
-	msg := ""
+	svcEvents := make([]corev1.Event, 0)
 	for _, e := range events.Items {
 		if e.InvolvedObject.Name == svcName {
-			msg = e.Message
-			break
+			svcEvents = append(svcEvents, e)
 		}
 	}
-	if msg == "" {
+	if len(svcEvents) == 0 {
 		return nil, errors.New("service doesn't be assigned node")
 	}
 
+	sort.Slice(svcEvents, func(i, j int) bool {
+		return svcEvents[i].LastTimestamp.Before(&svcEvents[j].LastTimestamp)
+	})
+
+	msg := svcEvents[len(svcEvents)-1].Message
```

Comment:
can we sort the other way around and pick svcEvents[0]? I think it'd be clearer

Reply:
done